samples/test_is_gt.yml

```diff
@@ -2,12 +2,12 @@ tests_include:
   - check_chassis_fpc
 
 check_chassis_fpc:
-  - command: show chassis fpc
+
```

Comment:
Same comment as above here also. Though it depicts correct usage, this may confuse some readers